NoahSaxophone.md

```diff
@@ -0,0 +1,25 @@
+---
+name: "@NoahSaxophone"
+project: "ADS-B Receiver"
+---
+
+# ADS-B Receiver
+
+## Summary
+
+I am going to build an ADS-B receiver so I can track a plane's location in the sky. Right now, my plan is to become a pilot when I turn 17. I've joined the Royal Canadian Air Cadets and hope to get my license through them. I've wanted to do this project for a long time now, but I couldn't find a raspberry pi that was cheap enough to do so.
+
+## Plan
+
+After purchasing the components, my plan is to put them together. Next, I will download FlightRadar's Pi24 software and put it on the raspberry pi. Then, I will set it up and (hopefully) be able to see each plane's telemetry. Finally, I will design and 3-D print a case for everything to allow it to be outside. I know it sounds simple, but I find it fascinating that anyone can just scan the sky for planes.
```

Comment:
Would you need filament to print the case? I don't see it in your budget, so I assume you have that on hand.

Reply:
Yes, I already have that. Would I need to include that in the budget?

Reply:
Yes please! Just mention you already own it.